app/presenters/wpcc/message_presenter.rb

```diff
@@ -1,5 +1,6 @@
 module Wpcc
   class MessagePresenter < Presenter
+    include ActionView::Helpers::NumberHelper
```

Comment:
You already have the helpers on the view context. Can you remove this and called `view_context.number_with_delimiter`?